fdbserver/DataDistribution.actor.cpp

```diff
@@ -3550,6 +3568,11 @@ ACTOR Future<Void> storageServerTracker(
 			}
 			lastIsUnhealthy = status.isUnhealthy();
 
+			if(status.isTooFarBehind != server->ssVersionTooFarBehind.get()) {
+				TraceEvent("SSVersionHEREE", self->distributorId); // TODO: Remove
```

Comment:
Does this ever happen? Can you make the trace event as SevError and run correctness test on this PR, so that we know if it ever happens. I doubt it ever happens.

Reply:
Theoretically it is possible to hit this, right? Or do you think it is not possible to hit this at all.

Reply:
I think the `status ` was updated above and there is no wait in between. That's why I guess it won't happen. (maybe correctness can tell if I'm wrong. :))

Reply:
There is at least  one wait in between nested under an if condition. That's why i added the check.